### DISCUSSION TITLE
Migrate from MergeTree to ReplicatedMergeTree in multi-disk configuration

### DISCUSSION THREAD

**Discussion by Firm** [ClickHouse/ClickHouse#33400] [Q&A]
ClickHouse documentation has example of how to migrate from MergeTree engine to ReplicatedMergeTree for particular table. However, it doesn't provide much information how to do this in multi-disk configuration (when volume has more than one disk). Does anyone have ideas how it's possible?

**Comment by den-crane** [comment]
I suggest to use `alter table attach partition from` instead of manipulations with files.

**Reply by Firm** [reply]
Could you please direct me to manual that describes these things for multi-disk configuration? Original documentation seems to have single-disk configuration in mind.
